cli-playground/devfile-parser
# Setup devfile parser library

Parser library implementation is functional

Currently it only outputs a devfile object instead of generating kubernetes deployable objects, but that could be the next step depending on how we decide to do it 

steps to check execution:

- run ```go build main.go```
- run ```./main```

You should see the value of dockerfileLocation present in dockerfile component of included devfile.yaml printed on the terminal
